query-connector/src/app/(pages)/landingPage/landingPage.tsx

```diff
@@ -5,15 +5,37 @@ import {
   ProcessListItem,
   ProcessListHeading,
 } from "@trussworks/react-uswds";
-import Link from "next/link";
 import Image from "next/image";
 import styles from "./landingPage.module.scss";
+import { signIn, useSession } from "next-auth/react";
+import { useContext, useEffect } from "react";
+import { DataContext } from "@/app/shared/DataProvider";
+import { isAuthDisabledClientCheck } from "@/app/utils/auth";
 
 /**
  * The landing page for the TEFCA Viewer.
  * @returns The LandingPage component.
  */
 export default function LandingPage() {
+  const { status } = useSession();
+  const ctx = useContext(DataContext);
+  const isAuthDisabled = isAuthDisabledClientCheck(ctx?.runtimeConfig);
+
+  useEffect(() => {
+    if (isAuthDisabled || status === "authenticated") {
+      window.location.href = "/query";
+    }
+  }),
+    [isAuthDisabled];
```

Comment:
I think we want this array to go inside the useEffect on 28. As is, don't think this is doing anything